deploy/1-deploy-oracles.ts

```diff
@@ -19,7 +18,7 @@ const ADDRESSES = {
   },
 };
 
-const func: DeployFunction = async function ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) {
+module.exports = async function ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) {
```

Comment:
why are we not using hardhat-deploy function?

Reply:
I have reverted back to use `DeployFunction` interface.